model-optimizer/mo/main.py

```diff
@@ -270,16 +283,16 @@ def emit_ir(graph: Graph, argv: argparse.Namespace):
         # This try-except is additional reinsurance that the IE
         # dependency search does not break the MO pipeline
         try:
-            if not argv.legacy_ir_generation and find_ie_version(silent=True):
+            if not argv.legacy_ir_generation and ie_is_available:
                 path_to_offline_transformations = os.path.join(os.path.realpath(os.path.dirname(__file__)), 'back',
                                                                'offline_transformations.py')
                 status = subprocess.run([sys.executable, path_to_offline_transformations,
                                          "--input_model", orig_model_name,
-                                         "--framework", argv.framework], env=os.environ, timeout=10)
+                                         "--framework", argv.framework,
+                                         "--transform", argv.transform], env=os.environ)
                 return_code = status.returncode
-                if return_code != 0 and not argv.silent:
-                    log.error("offline_transformations return code {}".format(return_code), extra={'is_warning': True})
         except Exception as e:
+            return_code = "failed"
```

Comment:
It looks a bit strange that you use string value as a return code

Reply:
Discussed offline. Closing.